<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperatorsDlg</class>
 <widget class="QDialog" name="OperatorsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>655</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>User profiles</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lblOperators">
            <property name="text">
             <string>Operators</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnNew">
            <property name="text">
             <string>&amp;New</string>
            </property>
            <property name="icon">
             <iconset resource="../qHostMonitor.qrc">
              <normaloff>:/img/new.png</normaloff>:/img/new.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCopy">
            <property name="text">
             <string>&amp;Copy</string>
            </property>
            <property name="icon">
             <iconset resource="../qHostMonitor.qrc">
              <normaloff>:/img/action/copy.png</normaloff>:/img/action/copy.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRename">
            <property name="text">
             <string>&amp;Rename</string>
            </property>
            <property name="icon">
             <iconset resource="../qHostMonitor.qrc">
              <normaloff>:/img/action/properties.png</normaloff>:/img/action/properties.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelete">
            <property name="text">
             <string>&amp;Delete</string>
            </property>
            <property name="icon">
             <iconset resource="../qHostMonitor.qrc">
              <normaloff>:/img/trash.png</normaloff>:/img/trash.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listUserProfiles"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>15</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="verticalSpacing">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lblID">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>ID</string>
              </property>
              <property name="buddy">
               <cstring>cmbID</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cmbID">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lblFullName">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Full name</string>
              </property>
              <property name="buddy">
               <cstring>cmbFullName</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="cmbFullName">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lblPassword">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Password</string>
              </property>
              <property name="buddy">
               <cstring>editPassword</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="editPassword">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lblConfirm">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Confirm</string>
              </property>
              <property name="buddy">
               <cstring>editConfirm</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="editConfirm">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="verticalSpacing">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lblStatus">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Status</string>
              </property>
              <property name="buddy">
               <cstring>cmbStatus</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cmbStatus">
              <item>
               <property name="text">
                <string>Account enabled</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Account disabled</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lblEmail">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>E-Mail</string>
              </property>
              <property name="buddy">
               <cstring>cmbEmail</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="cmbEmail">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lblPager">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Pager</string>
              </property>
              <property name="buddy">
               <cstring>cmbPager</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="cmbPager">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lblICQ">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>ICQ</string>
              </property>
              <property name="buddy">
               <cstring>cmbICQ</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="cmbICQ">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>15</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lblAcceptRemoteConnection">
              <property name="text">
               <string>Accept remote connections from</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listAcceptRemoteConnection">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lblUserRights">
              <property name="text">
               <string>User has rights to</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listUserRights">
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <item>
               <property name="text">
                <string>add tests/folders</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>configure tests/folders</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>remove tests/folders</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>disable/enable tests</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>reset statistic</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>start/stop monitoring</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>change options</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>manage schedules</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>manage mail profiles</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>manage color profiles</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>manage report profiles</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>manage action profiles</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>manage user profiles</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>acknowledge test status</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>pause/resume tests</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>edit GUI options for the account</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>edit GUI options for ALL accounts</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
              <item>
               <property name="text">
                <string>view test settings (read-only)</string>
               </property>
               <property name="checkState">
                <enum>Unchecked</enum>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="chkRccLimit">
            <property name="text">
             <string>Limit number of RCC connections for the account to</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinRccLimit">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblConnection">
            <property name="text">
             <string>connection</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbAllFoldersAnyHml">
          <property name="text">
           <string>All folders in any HML list</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rbFolder">
            <property name="text">
             <string>The following folder only</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editFolder">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnFolder">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../qHostMonitor.qrc">
              <normaloff>:/img/foldersTree.png</normaloff>:/img/foldersTree.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="rbNoAccessToHml">
          <property name="text">
           <string>No access to this HML list</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="bottomMargin">
      <number>470</number>
     </property>
     <item>
      <widget class="QDialogButtonBox" name="btnBoxUserProfiles">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actAddIP">
   <property name="text">
    <string>Add IP address...</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actRemoveIP">
   <property name="text">
    <string>Remove Item</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qHostMonitor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnBoxUserProfiles</sender>
   <signal>rejected()</signal>
   <receiver>OperatorsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
